### PR TITLE
Make sure the test start with the default ltpa configuration

### PR DESCRIPTION
**Pull request by utle**
#build

### CODE REVIEW

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#run-libby-bot

**Comment by utle** [comment]
#run-libby-bot

File: dev/com.ibm.ws.security.authentication.filter_fat/fat/src/com/ibm/ws/security/authentication/filter/fat/DynamicAuthFilterTest.java [hunk 1]
```diff
@@ -35,261 +35,264 @@
...
     /**
-     * Test description:
-     * - Reconfigure the server to remove the id of the authfilter
-     * - Do a sucessful a servlet call since all resources will be protected.
-     * - Reconfigure the server to add elements on the authfilter.
-     * - do another sucessfull a servlet call.
-     * Expected results:
-     * - 1) A 200 should be received since all resources will be protected.(but expect the error message to pop out).
-     * - 2) Another 200 since we use a valid configuration.
+     * Test description: - Reconfigure the server to set a malformed IP in the
```
Comment (Zech-Hein) [inline dev/com.ibm.ws.security.authentication.filter_fat/fat/src/com/ibm/ws/security/authentication/filter/fat/DynamicAuthFilterTest.java]: looks like eclipse messed up the formatting on these comments

**Review by Zech-Hein — APPROVED** [review]

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#run-libby-bot

**Pull request merged**